lib/sbconstants/cli.rb

```diff
@@ -65,14 +66,21 @@ def refute_key_collisions
     #
     # A constant key can potentially exist in many files so locations is a collection
     def parse_storyboards
-      Dir["#{options.source_dir}/**/*.storyboard"].each do |storyboard|
+      Dir["#{options.source_dir}/**/*.storyboard"].each_with_index do |storyboard, storyboard_index|
+
+        filename = File.basename(storyboard, '.storyboard')
+        storyboards << filename
+
+        constants[filename] << Location.new('storyboards', nil, storyboard, filename, storyboard_index + 1)
```

Comment:
I think this should be changed to `Location.new('storyboardNames', nil, storyboard, filename, storyboard_index + 1)` to decrease name collisions